src/api/apiUtils.js

```diff
@@ -9,15 +9,15 @@ export async function handleResponse(response) {
   const error = await response.json();
   log.debug('handleResponse error ---', response.status, error);
 
-  if (response.status === 422) {
+  // pass along user errors
```

Comment:
I'm a little concerned about splitting out error handling into 4xx and 5xx like this.
It puts the onus on the caller to implement two separate ways of catching errors.
A 4xx doesn't necessarily mean a _user_ error, it could just be caused by a bug in the client or a mismatch with the API.

I think my preference would be to always throw the error object (status and message) and expect the caller to wrap the call in a `try...catch`, but I'm happy to be challenged.
If the caller knows what to do with the status code, it can behave accordingly. Otherwise, it rethrows and a higher-level `catch` can be responsible for showing an alert when something truly unhandled happens.
What do you think?

Reply:
@xxGreyscale may have an opinion on this with the work on #473.

Reply:
I'll give it a go!